libminifi/test/unit/RegexUtilsTests.cpp

```diff
@@ -0,0 +1,59 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "../TestBase.h"
+#include "utils/RegexUtils.h"
+#include <string>
+#include <vector>
+
+using org::apache::nifi::minifi::utils::RegexUtils;
+
+TEST_CASE("TestRegexUtils::single_match", "[regex1]") {
+    std::string pat = "Speed limit 130 | Speed limit 80";
+    std::string rgx1 = "Speed limit ([0-9]+)";
+    std::vector<RegexUtils::Mode> mode = {RegexUtils::Mode::EXTENDED, RegexUtils::Mode::ICASE};
+    RegexUtils r1(rgx1, mode);
+    REQUIRE(r1.canMatch(pat));
+    auto ret = r1.getResult();
+    std::vector<std::string> ans;
```

Comment:
In C++11 you can init this as
```
std::vector<std::string> ans = {"Speed limit 130", "130"};
```

Reply:
lol, I don't know why I didn't use that albeit I already used for initializing regex mode vector